### PR TITLE
Feature/split into first modules

### PR DESCRIPTION
**Pull request by beyer-johannes**
Finish #23 
and first steps for #41

### CODE REVIEW

File: packages/ant-auth/prismerge.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"core":{"inputs":["/home/hannes/projects/antify/antify/packages/ant-database/src/runtime/core/schema.prisma","/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/core/schema.prisma","/home/hannes/projects/antify/antify/packages/ant-auth/src/runtime/core/schema.prisma"],"fragments":["/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/core/schema.prisma"],"output":"/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/core/schema.merged.prisma"},"tenant":{"inputs":["/home/hannes/projects/antify/antify/packages/ant-database/src/runtime/tenant/schema.prisma","/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/tenant/schema.prisma"],"output":"/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/tenant/schema.merged.prisma"}}
```
Comment (ThomasKro) [inline packages/ant-auth/prismerge.json:1]: This probably should not be commited, seems to me that it is generated and there are lokal pathes in here.

File: packages/ant-database/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (ThomasKro) [inline packages/ant-database/.editorconfig:1]: isn't the global .editorconfig enough? I thought that you don't need them in sub directories if nothing changes

File: packages/ant-auth/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (ThomasKro) [inline packages/ant-auth/.editorconfig:1]: isn't the global .editorconfig enough? I thought that you don't need them in sub directories if nothing changes

**Review by ThomasKro — DISMISSED** [review]

File: packages/ant-auth/.editorconfig (hunk 3 above)
Comment (beyer-johannes) [reply packages/ant-auth/.editorconfig]: You're right. Removed

**Pull request merged**